services/onboarding_service.rb

```diff
@@ -80,7 +75,11 @@ def local_configuration_repo_exists?
 
     # @return [Boolean]
     def remote_configuration_repository_valid?
-      return Services.configuration_repository_service.configuration_repository_valid?
+      if Services.configuration_repository_service.respond_to?(:configuration_repository_valid?)
```

Comment:
Weird, why do we need to check this?

Reply:
I have encountered cases when `configuration_repository_service` was nil, but couldn't track the source of this issue, so I made this extra safe call just in case. Don't know if this is the best way to proceed tho.

Reply:
Ohhh weird, I think we should leave this check off then. That way when we encounter it we can fix it. If we can't find it for some reason, we can always follow up and add it back

Reply:
Done! Rebasing tomorrow😁

Reply:
🎉